scapy/tools/automotive/obdscanner.py

```diff
@@ -0,0 +1,163 @@
+#! /usr/bin/env python
+
+# This file is part of Scapy
+# See http://www.secdev.org/projects/scapy for more information
+# Copyright (C) Andreas Korb <andreas.korb@e-mundo.de>
+# Copyright (C) Friedrich Feigel <friedrich.feigel@e-mundo.de>
+# This program is published under a GPLv2 license
+
+from __future__ import print_function
+
+import getopt
+import sys
+import signal
+
+import scapy.modules.six as six
+from scapy.config import conf
+from scapy.consts import LINUX
+
+if six.PY2 or not LINUX:
+    conf.contribs['CANSocket'] = {'use-python-can': True}
+
+from scapy.contrib.isotp import ISOTPSocket                 # noqa: E402
+from scapy.contrib.cansocket import CANSocket, PYTHON_CAN   # noqa: E402
+from scapy.contrib.automotive.obd.obd import OBD            # noqa: E402
+from scapy.contrib.automotive.obd.scanner import obd_scan   # noqa: E402
+
+
+def signal_handler(sig, frame):
+    print('Interrupting scan!')
+    sys.exit(0)
+
+
+def usage():
+    print('''usage:\tobdscanner [-i|--interface] [-c|--channel] [-s|--source] [-d|--destination] [-b|--bitrate] 
+                                [-h|--help] [-t|--timeout] [-r|--supported] [-u|--unsupported] [-v|--verbose]\n
+    Scan for all possible obd service classes and their subfunctions.\n
+    optional arguments:
+    -c, --channel               python-can channel or Linux SocketCAN interface name
+    -s, --source                ISOTP-socket source id (hex)
```

Comment:
could you please add default values for source and destination

Reply:
I've set the destination to the OBD-Broadcast-Domain.
Should they be non-mandatory, then?

Reply:
Yes, I would like to have source and destination as optional arguments. This makes the default usage faster

Reply:
Ok.